WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/WebView.swift

```diff
@@ -14,6 +14,9 @@ struct WebView: UIViewRepresentable {
         }
     }
 
+    let webView: WKWebView = WKWebView()
+    let progressView: WebProgressView = WebProgressView()
+
     let url: URL
```

Comment:
TIL about the `WebProgressView`!

Let's keep the properties private as they don't need to be accessed from outside.

We can also omit the type declaration or use the shorthand for the initializers.

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/pull/11501/commits/1c97497a0a41d5adee4f0541ae88800977ea37b7